apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -218,14 +210,14 @@ <h6>Downloads</h6>
                 </ul>
 
             </div>
-            <div class="col-sm-2 logo-hide-ipad">
+            <div class="col-sm-6">
                 <div class="giddh-logo">
-                    <img src="assets/images/giddh-logo.svg">
+                    <a [routerLink]="['pages/home']"> <img src="assets/images/giddh-white-logo.svg"> </a>
```

Comment:
WHAT YOU HAVE DONE HERE ?

Reply:
added giddh logo and onClick redirect on home page as per discussion with Gourav